languages/en.ini

```diff
@@ -1119,7 +1122,7 @@ Requests = "Requests"
 Reserves = "Reserves"
 Reserves Search = "Reserves Search"
 Reserves Search Results = "Reserves Search Results"
-reset_filters_button = "Reset Filters"
+reset_filters_button = "Reset All Filters"
```

Comment:
The "reset filters" button can be a bit strange, because it resets filters to their default state, based on configuration. In most cases, this means removing all filters, but in situations where visible filters are configured to appear by default, the reset button will restore the settings to the initial state... so it can potentially add filters as well as remove them. I'm not sure if changing the wording of the button helps, and it might make things worse in some cases. Would it be better to leave this to local customization?

Reply:
Oh, I wasn't aware of that. Sure, leave it as it was then!